### DISCUSSION TITLE
Ability to hide labels on argon counter text components

### DISCUSSION THREAD

**Discussion by Joehuu** [ppy/osu#25457] [Ideas]
<img width="230" alt="Screenshot 2023-11-14 at 1 01 00 PM" src="https://github.com/ppy/osu/assets/35318437/f638d968-f8e7-4e5e-a178-6cbda95a70e0">

For example the accuracy text here. I have the WIP changes here: https://github.com/ppy/osu/compare/master...Joehuu:osu:argon-hide-counter-labels

Will continue polishing and add localisation if accepted.

**Comment by peppy** [comment]
sure
